Week 04/Lecture 07/Assignment 02/assignment2/src/main/java/example/lecture7/assignment2/config/AppConfig.java

```diff
@@ -0,0 +1,9 @@
+package example.lecture7.assignment2.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = "example.lecture7.assignment2.service")
```

Comment:
@ComponentScan it can be used when we want to specifiy base package of the annotated class or when the folder in different folder. Actually when we don't need to use @ComponentScan if all the class is in the same pacakage. For this case @ComponentScan can be removed by remove @ComponentScan from class AppConfig and change the main function, scans default package. Done revised the code.